src/logic/safe/transactions/gasNew.ts

```diff
@@ -25,33 +28,40 @@ const estimateDataGasCosts = (data) => {
   return data.match(/.{2}/g).reduce(reducer, 0)
 }
 
-export const estimateTxGasCosts = async (safeAddress, to, data, tx?: any, preApprovingOwner?: any) => {
+export const estimateTxGasCosts = async (
+  safeAddress: string,
+  to: string,
+  data: string,
+  tx?: Transaction,
+  preApprovingOwner?: string,
+): Promise<number> => {
   try {
     const web3 = getWeb3()
     const from = await getAccountFrom(web3)
-    const safeInstance: any = new web3.eth.Contract(GnosisSafeSol.abi as any, safeAddress)
+    const safeInstance = (new web3.eth.Contract(
+      (GnosisSafeSol.abi as unknown) as AbiItem,
+      safeAddress,
+    ) as unknown) as GnosisSafe
     const nonce = await safeInstance.methods.nonce().call()
     const threshold = await safeInstance.methods.getThreshold().call()
-
-    const isExecution = (tx && tx.confirmations.size === threshold) || !!preApprovingOwner || threshold === '1'
+    const isExecution = (tx && tx.confirmations.size === Number(threshold)) || !!preApprovingOwner || threshold === '1'
```

Comment:
why not optional chaining instead?

```suggestion
    const isExecution = (tx?.confirmations.size === Number(threshold)) || !!preApprovingOwner || threshold === '1'
```